api/v1beta1/flinkcluster_types.go

```diff
@@ -210,6 +210,9 @@ type JobManagerSpec struct {
 	// Sidecar containers running alongside with the JobManager container in the
 	// pod.
 	Sidecars []corev1.Container `json:"sidecars,omitempty"`
+
+	// JobManager Deployment pod template annotations.
+	PodTemplateAnnotations map[string]string `json:"podTemplateAnnotations,omitempty"`
```

Comment:
I think we can simplify the name, s/PodTemplateAnnotations/PodAnnotations?

Reply:
Or maybe just call it Annotations.

Reply:
I feel `PodAnnotations` is better so I've changed. I intentionally avoid `Annotations`, I think it'll 
 have misleading it means annotations for `Deployment`, not pod template spec. Anyway how about this?